src/docs/product/issues/performance-issues/index.mdx

```diff
@@ -43,3 +43,5 @@ Performance issues currently have the following limitations:
 - Performance issues only support simple ignore rules
 
 We're working on removing some or all of these limitations.
+
+<!-- which of these limitations can be removed for GA -->
```

Comment:
@gggritso

Reply:
You can remove "You cannot create alerts for performance issues" and "Performance issues only support simple ignore rules"